### ISSUE TITLE
Eclipse Plugin "use implicit imports for annotation classes" not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#210]
```
Running the nullness checker on the example below doesn't report any problems 
and generates an error in the eclipse log.

public class Test {
    public Object getNonNull() {
        return getNull();
    }
    public /*@Nullable*/ Object getNull() {
        return null;
    }
}

from eclipse .log:

Message was: cannot find symbol
    public /*@Nullable*/ Object getNull() {
  symbol:   class Nullable
  location: class Test line was 9

In the preferences, I have "Use implicit imports for annotation classes" 
checked.

If I set "Additional compiler parameters" to
-J-Djsr308_imports=checkers.nullness.quals.*

then the nullness checker works as expected and no error is logged to eclipse 
.log


```

Original issue reported on code.google.com by `trask.st...@gmail.com` on 3 Mar 2013 at 2:54
